airgun/entities/computeprofile.py

```diff
@@ -31,6 +35,18 @@ def delete(self, entity_name):
         view.table.row(name=entity_name)['Actions'].widget.fill('Delete')
         self.browser.handle_alert()
 
+    def list_computeprofiles(self, entity_name, computeprofile_name):
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.compute_profiles.table.row(compute_profile=computeprofile_name)[
```

Comment:
Maybe i'm missing something, but this method is invalid according to ui. On compute profile details page there's no "Compute profiles" tab (actually, no tabs at all) and no table with compute profiles, only table with compute resources. Not to mention such "list_" method isn't really needed, as you can read entity to get table elements.

Reply:
I named the method wrong way. Basically, it follows these steps 
1. Infrastructure -> compute resources
2. Select the created ec2 CR.
3. Click Compute Profile tab.
4. Select (3-Large) and submit.

So you can't find  "Compute profiles" tab in compute profiles details page. I had this piece of code only in compute resource entity but then I thought from a comment above that this should be in compute profile entity too 
(session.compute_resource.do_main_part_of_functionality
session.compute_profile.your_method() for assert or whatever you need).

Reply:
ah, i've just noticed compute profile's 'Edit' step goes through compute resource page. I'd expect 'edit' step just opens entity regularly, without extra navigation, maybe it's worth renaming this 'Edit' to smth else.
But anyway i'm not sure we need these helpers as ComputeResourceEntity already has it's list/update/read_computeprofiles helpers and those are in the right place as for me. @oshtaier could you please clarify your comment and share your vision on how we should place them?

Reply:
Yes, @abalakh is absolutely right. There are no references for compute profiles after `view = self.navigate_to(self, 'Edit', entity_name=entity_name)`